raster/raster-layer-geopackage/README.md

```diff
@@ -1,23 +1,34 @@
-# Raster Layer GeoPackage
+# Raster layer (GeoPackage)
```

Comment:
Should we be adding the parentheses here if they weren't there before?

Reply:
yes: parentheses are ok to add here since they won't affect the samples names in code